src/store/store.ts

```diff
@@ -0,0 +1,6 @@
+import { configureStore } from "@reduxjs/toolkit";
+
+export const store = configureStore({
+  reducer: {},
+  devTools: true,
```

Comment:
Need to make it true only for dev environment, it's usually process.env.NODE_ENV === 'development' , but I'd double check how it's handled in vite

Reply:
Upd.
`import.meta.env.DEV: {boolean} whether the app is running in development (always the opposite of import.meta.env.PROD) `

Reply:
done